crates/nu-command/tests/commands/headers.rs

```diff
@@ -115,8 +110,7 @@ fn headers_invalid_column_type_duration() {
 
 #[test]
 fn headers_invalid_column_type_binary() {
-    let actual = nu!(
-    cwd: "tests/fixtures/formats", pipeline(
+    let actual = nu!(pipeline(
         r#"
             [[a b]; [("aa" | into binary), 2], [3,4] ]
             | headers"#
```

Comment:
on such tests, i'm wondering if we could just remove the `pipeline` call by putting the command on the same line? e.g.
```diff
-    let actual = nu!(pipeline(
-        r#"
-            [[a b]; [("aa" | into binary), 2], [3,4] ]
-            | headers"#
+    let actual = nu!(r#"
+        [[a b]; [("aa" | into binary), 2], [3,4] ] | headers
+    "#
```

Reply:
I was hesitant to move more complex stuff to single line expressions both for reabability of the code and also as I tried to limit the impact on the line blame. The latter is probably not always possible but I thought keeping the multiline stuff as is would keep a higher chance to see the reason for the test directly. 

I think there is still some room for improvement to make the tests more readable but this was a relatively monotic sprint through the quickfix list.

Reply:
completely legit answer, just wanted to bring that to the table in case you'd forgotten it :wink: